spswarehouse/credentials.py.template

```diff
@@ -42,4 +42,16 @@ calpads_config = {
     "username": "",
     "password": "",
     "host": "https://www.calpads.org"
-}
\ No newline at end of file
+}
+
+canvas_config = {
+	"client_id": "250260000000000002",
+	"client_secret": "",
+	"host": "https://summitps.instructure.com",
```

Comment:
Might be useful to put a comment at the end of this line of: `# Beta instance = https://summitps.beta.instructure.com`